### PR TITLE
Bump version to 0.5

### PR DESCRIPTION
**Pull request by michaelwoerister**
r? @wesleywiser 

cc @jackpot51

### CODE REVIEW

**Comment by michaelwoerister** [comment]
I'd like to start using this version in `rustc` as soon as possible.

File: measureme/Cargo.toml [hunk 1]
```diff
@@ -18,4 +18,4 @@ rustc-hash = "1.0.1"
 parking_lot = "0.9"
 
 [target.'cfg(not(target_arch="wasm32"))'.dependencies]
-memmap = "0.6.0"
+memmap = "0.7"
```
Comment (wesleywiser) [inline measureme/Cargo.toml]: Do we also need to handle bumping `memmap` upstream in rustc? Or is it ok to have two different versions of `memmap` compiled in?

Comment (michaelwoerister) [reply measureme/Cargo.toml]: I think it's not a problem to have different versions (except for bloat), but I plan to also bump `memmap` on the `rustc` side in the same PR that bumps `rustc`'s `measureme` version.

**Comment by michaelwoerister** [comment]
Thanks for the quick review, @wesleywiser!

**Pull request merged**